tests/app/phase-tester-response/issue-responded/issue-responded.component.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { IssueRespondedComponent } from '../../../../src/app/phase-tester-response/issue-responded/issue-responded.component';
+import { Issue, STATUS } from '../../../../src/app/core/models/issue.model';
+import { ISSUE_WITH_EMPTY_DESCRIPTION } from '../../../constants/githubissue.constants';
+import { IssueComment } from '../../../../src/app/core/models/comment.model';
+import { USER_JUNWEI } from '../../../constants/data.constants';
+
+describe('IssuesPendingComponent', () => {
+  describe('.ngOnInit()', () => {
+    const issuesRespondedComponent = new IssueRespondedComponent();
+    issuesRespondedComponent.ngOnInit();
+    const DUMMY_TEAM = USER_JUNWEI.team;
+    const DUMMY_COMMENT: IssueComment = {
+      id: 1,
+      description: 'This is some description of an issue comment',
+    };
+    let dummyIssue: Issue;
+
+    beforeEach(() => {
+      // generate a well-formed dummy-issue
+      dummyIssue = Issue.createPhaseTeamResponseIssue(
+        ISSUE_WITH_EMPTY_DESCRIPTION,
+        DUMMY_TEAM
+      );
+    });
+
+    it('should set filter to return true for an issue with comment and is done', () => {
+      dummyIssue.status = STATUS.Done;
+      dummyIssue.issueComment = DUMMY_COMMENT;
+      expect(issuesRespondedComponent.filter(dummyIssue)).toBeTrue();
+    });
+
+    it('should set filter to return false for issues that are not done', () => {
```

Comment:
I think it would be clearer if we specify that it returns 'false for issues that are not done and have comments' in the description!

Reply:
Hmm I think you raise a good point that these tests are a little specific, but I would like to keep the purpose of this test to test for any issue that is "not done".

I can't really describe it properly but let me restructure the tests and get your input again!